test/fuzz/test_runner.py

```diff
@@ -22,6 +22,7 @@ def get_fuzz_env(*, target, source_dir):
         'UBSAN_OPTIONS':
         f'suppressions={source_dir}/test/sanitizer_suppressions/ubsan:print_stacktrace=1:halt_on_error=1:report_error_type=1',
         "ASAN_OPTIONS": "detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1",
+        'MSAN_SYMBOLIZER_PATH': os.environ.get('LLVM_SYMBOLIZER_PATH', "/usr/bin/llvm-symbolizer"),
```

Comment:
we don't have msan (memory sanitizer) enabled yet; impossible to test. maybe too early for this PR